Tactical/Overhead.cpp

```diff
@@ -11092,6 +11034,105 @@ void HandleTurncoatAttempt( SOLDIERTYPE* pSoldier )
 	}
 }
 
+void EscapeTimerCallback()
+{
+    const bool chanceToEscape = Chance(75);
+    bool escaped = false;
+    // Look for an escape direction for remaining mercs
+    std::vector<WorldDirections> possibleEscapeDirections{ NORTH, EAST, SOUTH, WEST };
+    std::random_shuffle(possibleEscapeDirections.begin(), possibleEscapeDirections.end());
+
+    for (auto direction : possibleEscapeDirections)
+    {
+        if (IsEscapeDirectionValid(direction) && chanceToEscape && gbWorldSectorZ == 0) // There is no escaping underground! For now..
+        {
+            escaped = true;
+            ScreenMsg(FONT_MCOLOR_LTYELLOW, MSG_INTERFACE, szPrisonerTextStr[STR_PRISONER_ESCAPE]);
+
+            JumpIntoEscapedSector(direction);
+            break;
+        }
+    }
+
+    if (!escaped)
+    {
+        ScreenMsg(FONT_MCOLOR_LTYELLOW, MSG_INTERFACE, szPrisonerTextStr[STR_PRISONER_NO_ESCAPE]);
+    }
+}
+
+void AttemptToCapturePlayerSoldiers()
+{
+#ifdef JA2UB
+    ScreenMsg(FONT_MCOLOR_LTYELLOW, MSG_INTERFACE, szPrisonerTextStr[STR_PRISONER_REFUSE_TAKE_PRISONERS]);

```

Comment:
we probably should just skip the whole surrender codepath if it's UB, no?

Reply:
This was the easiest way to deal with offering to surrender ourselves to the enemy. No need to disable or delete the button from UI, enemy simply wants you dead, always.

AI routines and quests related to POW are skipped via compiler switches.